roadmap/implementers-guide/src/runtime/router.md

```diff
@@ -10,16 +10,19 @@ Storage layout:
 /// Paras that are to be cleaned up at the end of the session.
 /// The entries are sorted ascending by the para id.
 OutgoingParas: Vec<ParaId>;
-/// Messages ready to be dispatched onto the relay chain.
+/// Messages ready to be dispatched onto the relay chain. The messages are processed in FIFO order.
```

Comment:
Maybe reframe this as "Dispatchable objects ready to be...".

There is no definition of `RawDispatchable` elsewhere in the guide, which is fine, but the comment should give some context.

Reply:
I spoke too soon, there is a definition of `RawDispatchable`.